mkdocs.yml

```diff
@@ -7,7 +7,7 @@ site_dir: site
 docs_dir: docs
 theme:
   name: readthedocs
-  navigation_depth: 3
+  navigation_depth: 4
```

Comment:
is this right?  necessary?

Reply:
I bumped it to make the menu navigation show the sub-items of more sections.

Reply:
cool - as long as it was not just leftover/extra.